### DISCUSSION TITLE
Using React Router on Deno

### DISCUSSION THREAD

**Discussion by ayoreis** [remix-run/react-router#8511] [Q&A] [answered]
👋 Hello, everybody!

I am trying to run React Router on Deno but these are the results importing in different ways:
- https://esm.sh: `error: Import 'https://cdn.esm.sh/v58/@babel/runtime@7.16.5/deno/helpers/esm/extends.js' failed: 500 Internal Server Error`
- https://dev.jspm.io: Doesn't give any error immediately but does when accessing its properties: `error: TS2339 [ERROR]: Property 'BrowserRouter' does not exist on type '{}'.`
- https://skypack.dev: The same as above.

How do I accomplish this?

**Comment by jacob-ebey** [comment]
You can find a repository with my preliminary Deno setup here: https://github.com/jacob-ebey/deno-remix.

**Reply by ayoreis** [reply]
Thanks! This is more complex than what I would like but I'm still gonna check it out better!

**Reply by ayoreis** [reply]
@jacob-ebey I only want to import _react-router_ not the entire _create-remix@latest_ setup. Do you know how to do that?

**Comment by ayoreis** [comment, marked as answer]
https://esm.sh has now fixed this with _v61_!!! (https://github.com/alephjs/esm.sh/issues/229#issuecomment-1004985430).
